src/composables/accounts.ts

```diff
@@ -39,12 +59,10 @@ const {
 } = createCallbackRegistry<(newAccount: IAccount, oldAccount: IAccount) => any>();
 
 const areAccountsRestored = ref(false);
+const isMnemonicRestored = ref(false);
+const passwordKey = ref<IKey | null>(null);
```

Comment:
I suggest to not use "password" word in such cases as this is actually not a password. The password is one of the properties. But the object itself is not a password. It's more like `encryptionKey`.
But the word "Key" is also misleading as the object has the `key` property. So at the end it's like a `encryptionData`.